sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/cases/EditCasePage.java

```diff
@@ -29,9 +29,12 @@
       By.cssSelector("#facilityOrHome input[checked] + label");
   public static final By PLACE_DESCRIPTION_INPUT = By.cssSelector("#healthFacilityDetails");
   public static final By EXTERNAL_ID_INPUT = By.id("externalID");
+  public static final By UUID_INPUT = By.id("uuid");
   public static final By DISEASE_INPUT = By.cssSelector("#disease input");
   public static final By USER_INFORMATION =
       By.cssSelector(".v-slot-view-header .v-slot-primary div");
   public static final By CASE_SAVED_MESSAGE = By.xpath("//*[contains(text(),'Case saved')]");
```

Comment:
A suggestion here for a CSS path: [class="v-Notification-caption"] or .v-Notification-caption

Reply:
I could't catch the message :)